ntbs-service/Pages/Notifications/OverviewPartials/_TreatmentEventsOverviewPartial.cshtml

```diff
@@ -48,14 +48,22 @@
             var treatmentEvents = Model.GroupedTreatmentEvents[episodeNumber];
             
             <div>
-                <h4 class="episode-grouping-overview-heading">Episode @episodeNumber</h4><span>@treatmentEvents.First().FormattedEventDate - @treatmentEvents.Last().FormattedEventDate </span>
+                <h4 class="episode-grouping-overview-heading">Episode @episodeNumber</h4>
+                @if (treatmentEvents.Last().TreatmentEventTypeIsOutcome)
```

Comment:
Does this want to be `if (Last().IsEpisodeEndingTreatmentEvent())`? Would require making the method public and declared as an extension

Reply:
Ah, this doesn't have changes from master and therefore I was using IsEpisodeEndingTreatmentEvent